room/room-compiler/src/main/kotlin/androidx/room/processor/MethodProcessorDelegate.kt

```diff
@@ -71,6 +76,31 @@ abstract class MethodProcessorDelegate(
             ).process()
         }
     }
+    fun isUsingMultipleConcurrencyPatterns(): Boolean {
+        val concurrencyTypes = listOf(
```

Comment:
We have a list just like this in [TransactionMethodProcessor](https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:room/room-compiler/src/main/kotlin/androidx/room/processor/TransactionMethodProcessor.kt;l=86-99), lets try and use it, likely moving it to a more accessible location, maybe in `javapoet_ext.kt`

Reply:
Didn't know about it! This list is not complete though. It's missing `LifecyclesTypeNames.COMPUTABLE_LIVE_DATA` and `ReactiveStreamsTypeNames.PUBLISHER`. Can I add them or they aren't there for a reason and I should ignore them?

Reply:
Yeah, you can totally add them, seems like we just missed them.

Reply:
Done